### PR TITLE
Added recommended setting validation_query_timeout to dbconfig.xml

### PR DESCRIPTION
**Pull request by mmolinac**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

At JIRA startup, you may receive a message about configuring `validation_query_timeout` in `dbconfig.xml` as a recommendation.
I've setup recommended values for MySQL and the same default values for Oracle and MSSQL, although they only recommend a default value for MySQL.

### Benefits

Follow Atlassian direction about MySQL configuration.

### Possible Drawbacks

None that I know of.

### Applicable Issues

Fixes https://github.com/idealista/jira_role/issues/20

### CODE REVIEW

**Comment by mmolinac** [comment]
Hmm. The Java version problem striked the test suite again, I'm afraid.

**Comment by jnogol** [comment]
It's freaking annoying. Could you please update the Changelog with this PR's change and https://github.com/idealista/jira_role/blob/master/molecule/default/requirements.yml#L3 to 3.4.2?

Thank you very much for your contributions :clap:

**Comment by mmolinac** [comment]
@jnogol Done!

File: molecule/default/requirements.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 - src: idealista.java_role
-  version: 3.3.0
+  version: 3.4.2
```
Comment (jperera) [inline molecule/default/requirements.yml:3]: Could you change it in tests/requirements.yml too? Thanks!

Comment (mmolinac) [reply molecule/default/requirements.yml:3]: Sure! Done.

**Review by jnogol — APPROVED** [review]

**Pull request merged**